lib/remap-lang/grammar.pest

```diff
@@ -41,7 +41,8 @@ operator_not            = { "!" }
 
 // Paths -----------------------------------------------------------------------
 
-path             = ${ ("." ~ path_segment)+ }
+path             = ${ path_root | ("." ~ path_segment)+ }
+path_root        = ${ ". " }
```

Comment:
I think `path_root = ${"." ~ (!"=")?}` might do it.

Reply:
This unfortunately isn't sufficient. However, I am about to put up a PR that makes this better (although still not 100% correct).

I'll get this one in as-is, and have the other PR iterate on it further.